<launch>

<arg name="pkg_name" value="led_interpreter"/>
<arg name="veh"  doc="Name of vehicle. ex: megaman"/>
<arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />
<arg name="protocol_file_name" default="LED_protocol"/>
<arg name="protocol_node_name" default="led_emitter_node"/>
<arg name="location_config" default="location_config"/>
<arg name="node_name" default="led_interpreter_node"/>
<arg name="intersectionType" />

<!-- <arg name="testing" default="false" doc="using for test purposes." /> -->

<node ns="$(arg veh)"   pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" required="true">
            <rosparam command="load" file="$(find led_interpreter)/config/$(arg protocol_node_name)/$(arg protocol_file_name).yaml"/>
            <rosparam command="load" file="$(find led_interpreter)/config/$(arg node_name)/$(arg location_config).yaml"/>
            <param name="intersectionType" value="$(arg intersectionType)"/>
</node>



<!-- Publication -->

<!-- Subscription -->


</launch>
